### PR TITLE
#14474: Fix OoO issues for Llama3 tests on CI

### PR DESCRIPTION
**Pull request by mtairum**
### Ticket
#14474

### Problem description
Reduce the LLama3 max sequence length for N150/N300 and for Llama3 small tests.
This fix is a temporary patch while we work on finishing functionality of Llama3. A follow-up PR will include automatic sequence length setup based on model params, chosen architecture and type of test.

### What's changed
- Re-add Llama3 1B/3B/8B/11B post-commit tests to CI.

### Checklist
- [x] [All-post-commit](https://github.com/tenstorrent/tt-metal/actions/runs/11871851940/job/33085015036)
  - This one was previously failing on CI-VM-120. Now passes.
- [x] [Single-chip tests](https://github.com/tenstorrent/tt-metal/actions/runs/11862115193/job/33061019935)
  - Llama3.2-1B failed, but I've since reduced the 16K seqlen to 4K.
- [x] [T3K tests](https://github.com/tenstorrent/tt-metal/actions/runs/11862118428)

### CODE REVIEW

**Comment by mtairum** [comment]
Waiting for the CI pipelines to pass. After they're green I'll merge and CI will be green again ❤️

**Review by cglagovichTT — APPROVED** [review]

**Pull request merged**
